template-helpers/emails.js

```diff
@@ -35,8 +35,8 @@ function emailBreachStats(args) {
 function getPreFxaUtmParams(serverUrl, content, userEmail) {
   const url = new URL(`${serverUrl}/oauth/init`);
   const utmParams = {
-    utm_source: "fx-monitor",
-    utm_medium: "fx-monitor-email",
+    utm_source: "fx-monitor-email",
+    utm_medium: "email",
```

Comment:
These seem like important changes, will this break any backwards compat with older things being logged?

Reply:
I'll check with @lesleyjanenorton about it. AFAIK no-one but ourselves is looking into our own email traffic sources.

Reply:
These won't get logged in our analytics since they're appended to the `/oauth/init` route which redirects directly to to FxA.  I think `utm_source` should remain `fx-monitor` to be consistent with other FxA pings and then perhaps we add `entrypoint: "pre-fxa-email"`.

Reply:
Oh dang, this is much ado about nothing now since we're already running the pre-FXA email blast.

So, I'm going to leave this as-is/was in `master`, so later when we look at GA reports and see the traffic from `utm_medium` of `fx-monitor-email` we'll see this is where it came from.